test/test_browser.py

```diff
@@ -2073,7 +2072,7 @@ def test_cubegeom_pre_regal(self):
     self.btest('third_party/cubegeom/cubegeom_pre.c', reference='third_party/cubegeom/cubegeom_pre.png', args=['-sUSE_REGAL', '-DUSE_REGAL', '-lGL', '-lSDL'])
 
   @requires_graphics_hardware
-  @requires_sync_compilation
+  @no_swiftshader
```

Comment:
What is this for?

Reply:
If you look at all the other `cubegeom_pre.c` around this one.. they are all marked as `@no_swiftshader`.    This particular one was never been run under swift shader before due to `@requires_sync_compilation` .. not that it is running, it fails, due to the swiftshader issue (I assume) so I added the same tag as the surrounding test.